loomi/regulariza-morador-client
# Feature/resident registration

## Feature
 - Resident Registration
---

**What I did:**

- Create Resident Registration Form and page
 - Abstract the logic of get nucleus of both Login and Registration Forms to a custom hook

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/135097107-83ea77b0-ab9e-4d85-b04f-050af8bdbf0a.png)
